### PR TITLE
Remove deprecated queued_retry processor

### PR DESCRIPTION
**Pull request by bogdandrutu**
Signed-off-by: Bogdan Drutu <bogdandrutu@gmail.com>

### CODE REVIEW

**Review by carlosalberto — APPROVED** [review]

**Comment by tigrannajaryan** [comment]
@bogdandrutu do all exporters in Core and contrib now use exporterhelper queue? Is queuedretry processor now longer needed by anyone?

**Comment by bogdandrutu** [comment]
@tigrannajaryan in core yes, in contrib the once that we are marked as owners yes. There may be others who doesn't, but I am not aware of any.

**Comment by tigrannajaryan** [comment]
> @tigrannajaryan in core yes, in contrib the once that we are marked as owners yes. There may be others who doesn't, but I am not aware of any.

Should we give a warning in Gitter that it is going away and give everyone in contrib 1-2 weeks to react and only remove after that?

**Comment by bogdandrutu** [comment]
> Should we give a warning in Gitter that it is going away and give everyone in contrib 1-2 weeks to react and only remove after that?

This was deprecated for more than 3 months, I don't know what you think we can achieve by waiting, definitely we can write to gitter.

File: cmd/schemagen/go.mod [hunk 1]
```diff
@@ -4,7 +4,9 @@ go 1.15
...
 require (
 	github.com/fatih/structtag v1.2.0
-	github.com/stretchr/testify v1.6.1
-	go.opentelemetry.io/collector v0.15.0
+	github.com/stretchr/testify v1.7.0
+	go.opentelemetry.io/collector v0.0.0-00010101000000-000000000000
 	gopkg.in/yaml.v2 v2.4.0
 )
+
+replace go.opentelemetry.io/collector => ../../
```
Comment (jrcamp) [inline cmd/schemagen/go.mod:12]: Did you mean to leave this?

Comment (bogdandrutu) [reply cmd/schemagen/go.mod:12]: Yes, to always build from head

**Comment by tigrannajaryan** [comment]
> > Should we give a warning in Gitter that it is going away and give everyone in contrib 1-2 weeks to react and only remove after that?
> 
> This was deprecated for more than 3 months, I don't know what you think we can achieve by waiting, definitely we can write to gitter.

OK, if you feel it is not an issue.

**Review by tigrannajaryan — APPROVED** [review]

**Pull request merged**